googleclient/README.md

```diff
@@ -0,0 +1,11 @@
+# Google Http Client - Feign Client
+
+This library is a feign [Client](https://github.com/OpenFeign/feign/blob/master/core/src/main/java/feign/Client.java) to use the java [Google Http Client](https://github.com/googleapis/google-http-java-client).
```

Comment:
Maybe phrase as:

This **module** is a feign Client **that uses** Google Http client

Reply:
Ah, missed that when bringing it over, will update!